tools/dep_updaters/update-acorn-walk.sh

```diff
@@ -17,7 +17,7 @@ DEPS_DIR="$BASE_DIR/deps"
 . "$BASE_DIR/tools/dep_updaters/utils.sh"
 
 NEW_VERSION=$("$NODE" "$NPM" view acorn-walk dist-tags.latest)
-CURRENT_VERSION=$("$NODE" -p "require('./deps/acorn/acorn-walk/package.json').version")
+CURRENT_VERSION=$("$NODE" "$NPM" --prefix './deps/acorn/acorn-walk/' pkg get version)
```

Comment:
Any reason for this? Using npm seems overkill.

Reply:
Not a particular reason, it feels more clean since its npm pkg get does exactly that

Reply:
I prefer the original one and it's faster.